src/apk.c

```diff
@@ -228,6 +235,9 @@ static const struct apk_option options_commit[] = {
 	{ 0x102, "clean-protected",	"Do not create .apk-new files in configuration dirs" },
 	{ 0x111, "overlay-from-stdin",	"Read list of overlay files from stdin" },
 	{ 0x113, "no-scripts",		"Do not execute any scripts" },
+	{ 0x117, "skip-hooks",		"Skip pre/post hook scripts (but not other scripts)" },
```

Comment:
Let's do `--no-commit-hooks` this in harmony with `no-scripts` and differentiates them better.

Reply:
agree, will fix